app/components/Configuration.tsx

```diff
@@ -164,31 +160,17 @@ export default class Configuration extends Component<Props> {
           </div>
 
           <div className={styles.header}>
-            {configuration.stage > 1 &&
-            configuration.stage < stages.length - 1 ? (
-              <>
+            {configuration.stage > 1 && configuration.stage < stages.length - 1 ? <>
```

Comment:
i like the old formatting much better, really wondering if prettier is set up correctly